### PR TITLE
Update rockbox.yml

### PR DESCRIPTION
**Pull request by xfouloux**
added login, semi-private, not public.

### CODE REVIEW

**Comment by garfield69** [comment]
i don't understand. I can browse for music and download the torrent metadata without login in, so what did I miss?

**Comment by xfouloux** [comment]
well, I think it is a bug, you should not be allowed to browse and download while not logged in, so I think it will be fixed soon, so better to add the login capability.

File: src/Jackett.Common/Definitions/rockbox.yml [hunk 1]
```diff
@@ -31,25 +45,27 @@
...
         selector: td a[href^="details.php?id="]
         attribute: href
-      comments:
+      comments: # link to the comments of the torrent
         selector: td a[href^="details.php?id="]
         attribute: href
       download:
         selector: td a[href^="download.php?id="]
         attribute: href
+        filters:
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/rockbox.yml]: please remove this debug statement

**Comment by xfouloux** [comment]
changed, my bad =)

**Pull request merged**